db/migrate/20220905200117_add_market_to_projects.rb

```diff
@@ -0,0 +1,5 @@
+class AddMarketToProjects < ActiveRecord::Migration[7.0]
+  def change
+    add_column :projects, :market, :integer
```

Comment:
Why not to also add for this migration `null: false` in order to have validation in database level?